var/spack/repos/builtin/packages/wxwidgets/package.py

```diff
@@ -42,6 +45,9 @@ def configure_args(self):
         spec = self.spec
         options = ["--enable-unicode", "--disable-precomp-headers"]
 
+        if "+opengl" in self.spec:
```

Comment:
```suggestion
        if self.spec.satisfies("+opengl"):
```

`in` can have some edge case hits in a spec. It's typically better to use `spec.contains()` instead.